internal/tasks/manifests.go

```diff
@@ -936,7 +987,24 @@ var securityInfoCheckSubmanifestInfoQuery = sqlext.SimplifyWhitespace(`
 		WHERE r.parent_digest = $1
 `)
 
-func (j *Janitor) doSecurityCheck(ctx context.Context, account keppel.Account, repo keppel.Repository, manifest keppel.Manifest, securityInfo *keppel.TrivySecurityInfo) (returnedError error) {
+func (j *Janitor) doSecurityCheck(ctx context.Context, inputSecurityInfo keppel.TrivySecurityInfo) (securityInfo keppel.TrivySecurityInfo, returnedError error) {
+	// we cannot use a pointer to the securityInfo because we want to return it into the returnChan
+	securityInfo = inputSecurityInfo
```

Comment:
Why though? The caller could just as well be rewritten from
```go
securityInfo, err := j.doSecurityCheck(ctx, inputSecurityInfo)
```
into
```go
err := j.doSecurityCheck(ctx, &securityInfo)
```
Then there is no risk of accidentally returning an empty `securityInfo` instead of the original one.